src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -372,6 +375,8 @@ void GcsActorManager::HandleGetNamedActorInfo(
   } else {
     reply->unsafe_arena_set_allocated_actor_table_data(
         iter->second->GetMutableActorTableData());
+    RAY_LOG(INFO) << "WANGTAO " << iter->second->GetState();
```

Comment:
please remove this log line or make this more informational

Reply:
@simon-mo Sorry for missing that, will remove it in https://github.com/ray-project/ray/pull/24204